apps/scratch/src/pages/__tests__/Solve.spec.tsx

```diff
@@ -150,7 +150,7 @@ test.describe("/solve/sessionId/taskId", () => {
   test("loads the initial task blocks", async ({ page: pwPage }) => {
     const page = new SolveTaskPage(pwPage);
 
-    await expect(page.blocksOfCurrentTarget).toHaveCount(2);
+    await expect(page.blocksOfCurrentTarget).toHaveCount(9);
```

Comment:
This `9` is a magic number that depends on the actual initial task.
It's repeated in a bunch of places, it'd be good to replace it with a constant.

Same goes for the other magic constants in this file, like `7` frozen blocks.